CoC updates

alesandroortiz opened the pull request:
- ~~Updates "religion" to "religious opinion"~~ Updates "religion" to "religion or lackthereof", as it's exclusionary of individuals who do not affiliate with a religion (whether that's because of their opinion or belief). ~~"Belief" is a charged word which implies actual belief in a religion in many societies, instead of both belief in a religion and belief in no religion. Using "opinion" seems like an appropriate to address this.~~
- Adds "language", as knowledge and use of native and fluent languages (and dialects, and accents) are often used as an identity marker by individuals (and by others when harassing).
- Adds "national origin", "ethnic origin", and "nationality", as these are very often used as identity markers by individuals. This also covers anti-Indigenous/Nativeness (which is already mentioned in this section.)
- Adds "relationships" to lifestyle choices and practices. Choices related to relationships (in the broadest sociological sense) are often a source of unwelcome comments, especially choices related to intimate relationships (e.g. intimate relationship status, involved parties in a relationship). Seems relevant to add here.

alesandroortiz committed:
Adds additional relevant identity markers to CoC

- Adds "ethnicity", as race is different than ethnicity
- Updates "religion" to "religious opinion", as it's exclusionary of individuals who do not affiliate with a religion (whether that's because of their opinion or belief). "Belief" is a charged word which implies actual belief in a religion in many societies, instead of both belief in a religion and belief in no religion. Using "opinion" seems like an appropriate to address this.
- Adds "language", as knowledge and use of native and fluent languages (and dialects, and accents) are often used as an identity marker by individuals (and by others when harassing)
- Adds "national origin", "ethnic origin", and "nationality", as these are very often used as identity markers by individuals. This also covers anti-Indigenous/Nativeness (which is already mentioned in this section.)

alesandroortiz committed:
Adds lifestyle choice to CoC

- Adds "relationships" to lifestyle choices and practices. Choices related to relationships (in the broadest sociological sense) are often a source of unwelcome comments, especially choices related to intimate relationships (e.g. intimate relationship status, involved parties in a relationship). Seems relevant to add here.

aredridel commented on coc.md:
```diff
@@ -24,8 +24,8 @@ layout: page
-					<li>Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, race, immigration status, religion, or other identity marker. This includes anti-Indigenous/Nativeness and anti-Blackness.</li>
-					<li>Unwelcome comments regarding a person&rsquo;s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment.</li>
+					<li>Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, race, national origin, ethnic origin, nationality, immigration status, language, religious opinion, or other identity marker. This includes anti-Indigenous/Nativeness and anti-Blackness.</li>
+					<li>Unwelcome comments regarding a person&rsquo;s lifestyle choices and practices, including those related to food, health, parenting, relationships, drugs, and employment.</li>
```
adding relationships :+1:

alesandroortiz committed:
Updates religion reference in harassment clause

After discussion in Slack #meta, this is clearer and more enforceable language.

alesandroortiz commented:
Updated based on Slack #meta discussion.

alesandroortiz committed:
Adds age to CoC

aredridel commented:
I'm +1
